NoMansLand/Assets/Scripts/RandomSpawner.cs

```diff
@@ -19,9 +19,10 @@ public class RandomSpawner : MonoBehaviour
     public int terrainZLength; // length of terrain in Z axis 
     public int itemXPos; // x position of item 
     public int itemZPos; // z position of item 
-    public const int cushionAmount = 50; // to ensure spawning does not happen at the edges (at the borders)
-
+    public const int cushionAmount = 70; // to ensure spawning does not happen at the edges (at the borders)
     private GameObject sampleObj;
+    public int spaceshipX; 
+    public int spaceshipZ; 
```

Comment:
Do we need these to be public? is anyone else using these values except for this file?